src/visualizer.js

```diff
@@ -239,9 +239,14 @@ global.visualizer.myRoomDatasDraw = function(roomName) {
 
   if (config.stats.summary && Memory.summary) {
     const highterQueue = _.chain(Memory.myRooms)
-      .map((roomName) => {
-        return {length: -Memory.rooms[roomName].queue.length, roomName: roomName};
-      }).sortBy((roomRet) => -Memory.rooms[roomRet.roomName].queue.length).value()[0];
+      .map((roomName) => ({
```

Comment:
I guess the `highterQueue` should be called `higherQueue`? - I know, not necessarily part of this PR, could be fixed.

Reply:
I assumed it had some deeper meaning. I would call it "longestQueue"